### PR TITLE
Make Function.prototype executable.

### PR DESCRIPTION
**Pull request by NeilFraser**

### CODE REVIEW

File: server/interpreter.js [hunk 1]
```diff
@@ -542,6 +546,7 @@ Interpreter.prototype.initObject = function(scope) {
  */
 Interpreter.prototype.initArray = function(scope) {
   var thisInterpreter = this;
+  this.ARRAY = this.createArray(this.OBJECT);
```
Comment (cpcallen) [inline server/interpreter.js]: Maybe add comment on line above here as in initDate, initRegExp etc.

File: server/interpreter.js [hunk 2]
```diff
@@ -289,6 +287,12 @@ Interpreter.prototype.initFunction = function(scope) {
   var FunctionConst = this.createNativeFunction(wrapper, this.FUNCTION);
   this.addVariableToScope(scope, 'Function', FunctionConst);
 
+  // Configure Function.prototype.
+  this.FUNCTION.nativeFunc = function() {};
+  this.FUNCTION.id = this.functionCounter_++;
+  this.setProperty(this.FUNCTION, 'length', 0,
+      Interpreter.READONLY_DESCRIPTOR);
```
Comment (cpcallen) [inline server/interpreter.js]: Why not just do

```JS
  this.FUNCTION = this.createNativeFunction(function() {}, this.OBJECT);
```
when creating it in initGlobalScope?

Comment (NeilFraser) [inline server/interpreter.js]: Confirmed.  That works.

File: server/interpreter.js (hunk 1 above)
Comment (NeilFraser) [inline server/interpreter.js]: Done.

**Pull request merged**